planning/surround_obstacle_checker/src/node.cpp

```diff
@@ -152,9 +152,6 @@ SurroundObstacleCheckerNode::SurroundObstacleCheckerNode(const rclcpp::NodeOptio
     p.surround_check_recover_distance =
       this->declare_parameter("surround_check_recover_distance", 2.5);
     p.state_clear_time = this->declare_parameter("state_clear_time", 2.0);
-    p.stop_state_ego_speed = this->declare_parameter("stop_state_ego_speed", 0.1);
-    p.stop_state_entry_duration_time =
-      this->declare_parameter("stop_state_entry_duration_time", 0.1);
```

Comment:
@satoshi-ota what reason did you delete this parameter? :pray:

Reply:
Sorry for late reply :pray: 

`vehicle_state_checker` has substitute param for `stop_state_ego_speed`, so I think this param is not necessary.

https://github.com/autowarefoundation/autoware.universe/blob/c88528d605f8e5acba7e54ad687d44c71b6334a4/common/motion_utils/src/vehicle/vehicle_state_checker.cpp#L39

Reply:
And, now that stop filter exists in localization module, I think this module doesn't have to check the elapse time after the vehicle stopped in order to prevent stop judge hunting. What do you think ?